proto/panacea/oracle/v2alpha2/oracle.proto

```diff
@@ -131,4 +133,27 @@ enum VoteOption {
 
   // VOTE_OPTION_INVALID
   VOTE_OPTION_NO = 2;
+}
+
+// UpgradeOracleInfo defines the info of oracle upgrade, which includes the target height of upgrade and unique ID of the new version of oracle
+message OracleUpgradeInfo {
+  option (gogoproto.equal) = true;
+
+  string unique_id = 1;
+
+  int64 height = 2;
+}
+
+// OracleRegistrationType defines of oracle registration type.
+enum OracleRegistrationType {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // UNSPECIFIED type
+  ORACLE_REGISTRATION_TYPE_UNSPECIFIED = 0;
+
+  // New oracle type
+  ORACLE_REGISTRATION_TYPE_NEW = 1;
+
+  // Upgrade oracle type
+  ORACLE_REGISTRATION_TYPE_UPGRADE = 2;
```

Comment:
```suggestion
  // For oracles newly added 
  ORACLE_REGISTRATION_TYPE_NEW = 1;

  // For oracles to be upgraded
  ORACLE_REGISTRATION_TYPE_UPGRADE = 2;
```
Good to have these types :) I just added a simple comment for comments :)

Reply:
Thank you!